release-process-documentation/documentation-guides/update-release-docs-new.md

```diff
@@ -0,0 +1,171 @@
+
+*   Generate reference docs and place them in kubernetes/website. See [PR 6366](https://github.com/kubernetes/website/pull/6366).
+*   Update reference page for feature gates. See [PR 6364](https://github.com/kubernetes/website/pull/6364).
+
+### Last minute PRs
+
+*   Triage: 
+    *   Decide what must go out with release, what can be published just after the release.
+    *   Determine how much effort will be required to review the doc(s).
+        *   Is it a complex feature needing specific reviewers?
+        *   Is it a long doc? Is it several docs?
+        *   Do you have edit permissions to help speed up the turnaround? If not, request that they check the [Allow edits from maintainers](https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/) checkbox.
+*   Get the PR author to help find tech reviewers.
+*   Ping the PR author to incorporate review feedback ASAP.
+*   Ask the release manager at the burndown meeting to help find appropriate tech reviewers.
+*   If not critical to initial release, un-merged PRs can be dealt with after the release.
+
+### Finalize for release
+
+*   Create a PR for the merge but do not merge yet, e.g. [PR #4094](https://github.com/kubernetes/kubernetes.github.io/pull/4094) for v1.7 release
+    *   Add tag _do-not-merge_
+    *   Add note and target date in the description, e.g.
+_DO NOT MERGE UNTIL KUBERNETES 1.7 IS RELEASED
+Target Date: 2017-06-29 @ 13:00 PDT_
+*   Update the site variables for version numbers:
+    *   Open a PR to modify [/_config.yml](https://github.com/kubernetes/kubernetes.github.io/blob/master/_config.yml) and change version variables to v1.8 and v1.8.0, e.g. [PR #4210](https://github.com/kubernetes/kubernetes.github.io/pull/4210/files).
+    *   Update [/update-imported-docs.sh](https://github.com/kubernetes/kubernetes.github.io/blob/master/update-imported-docs.sh), e.g. [PR #4210](https://github.com/kubernetes/kubernetes.github.io/pull/4210/files#diff-8696ecc75d1b25a0b2812f88aefea8c3).
+*   Make sure the auto-generated docs have been updated.
+    *   Check with Phillip Wittrock ([pwittroc@google.com](mailto:pwittroc@google.com)).
+    *   Run update-imported-docs.sh.
+*   Make sure the tutorials are imported from kubernetes/examples repo.
+    *   Run ./update-imported-tutorials.py
+*   Merge `master` back into the branch for the last release to create final snapshot of the last release, e.g. finalize `release-1.7` branch before `release-1.8` is merged into `master`.
+*   Add tags to `master`, last release, and next release branches to keep track of snapshot points, e.g. `snapshot-final-v1.6`, `snapshot-initial-v1.7`, etc.
+*   Update links for release announcement given by Kubernetes PM, currently Aparna Sinha ([apsinha@google.com](mailto:apsinha@google.com)), e.g. [Kubernetes.io Blog: 1.7 Release announcement](https://docs.google.com/a/google.com/document/d/1U8oNYK-baoF-ObIRFKEmCicDnrgFdXAz7Nfkatg_jUo/edit?usp=sharing).
```

Comment:
Aparna Sinha & Ihor Dvoretskyi (SIG-PM co-leads);
Natasha Woods & Kaitlyn Barnard (CNCF Marketing team);
and features and marketing leads/shadows (if they are different from the people, stated above).